draft-privacypass-rate-limit-tokens.md

```diff
@@ -611,8 +611,12 @@ compromise; see {{sec-considerations}} for additional about this channel.
 Before copying and forwarding the Client's TokenRequest request to the Issuer,
 the Attester validates the Client's stable mapping request as described in {{attester-anon-issuer-origin-id}}.
 If this fails, the Attester MUST return an HTTP 400 error to the Client.
-The Attester MAY also add additional context information, but MUST NOT add information
-that will uniquely identify a Client.
+The Attester MUST NOT add information that will uniquely identify a Client,
+or associate the request with a small set of possible Clients. Extensions
+to this protocol MAY allow Attesters to add information that can be used
+to separate large populations (numbering in the thousands or more), such
```

Comment:
Maybe leave the size of a "large population" unspecified instead of saying thousands. Valid population sizes sort of depends on the number of clients (a big proxy with millions of users, might want to say a large population is at least 50k, while a small proxy with thousands of users might want to allow smaller populations (for geolocation/caching/etc)).

Reply:
Fixed!